### PR TITLE
Build/split

### PR DESCRIPTION
**Pull request by SimonSchick**
Fixing tests with new carina version and split build!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,48 +1,3 @@
...
-# Optional eslint cache
-.eslintcache
-
-# Optional REPL history
-.node_repl_history
-
-/lib
-
-.vscode/
-yarn.lock
```
Comment (connor4312) [inline .gitignore]: plz keep the full list. The bytes are cheap, less churn and accidentally committed files that might appear in various workflows

Comment (SimonSchick) [inline .gitignore]: Most of these files should never appear in this repo anyways, which ones do you deem common?

Comment (connor4312) [inline .gitignore]: Generally I'd prefer to just keep all of em, even if we don't use them now. Less things to think about, and deal with if we do add something that uses them (like Istanbul)

**Comment by SimonSchick** [comment]
Resolved via Slack

**Pull request merged**